docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,137 @@
+Kqueen and Google cloud from scratch
```

Comment:
`Google cloud CLI operations with Kqueen`  or smth similar, coz imo this header is unclear

Reply:
Ok... Fixed...